src/modules/pages/EditPage/ApiItem.js

```diff
@@ -21,37 +23,52 @@ const propTypes = {
 class ApiItem extends Component {
     componentDidMount() {
         this.props.resetEndpoint();
+        this.props.fetchEndpointSchema();
         this.props.fetchEndpoint(this.props.location.pathname);
     }
 
-    submit = (values) => {
-        this.props.updateEndpoint(this.props.location.pathname, values);
-    };
+    fillSelected = (arr) => {
+        const selectedPlugins = arr.map(item => item.name);
+
+        this.props.fillSelected(selectedPlugins);
+    }
+
+    submit = values => {
+        const transformedValues = transformFormValues(values, true);
+        const plugins = transformedValues.plugins;
+        const selectedPlugins = this.props.selectedPlugins;
+
+        const addedPlugins = plugins.filter((plugin) => {
```

Comment:
maybe `const addedPlugins = plugins.filter((plugin) => selectedPlugins.indexOf(plugin.name) !== -1)`

Reply:
In my humble opinion, the current version is just easier to read :relieved:

Reply:
np, perhaps the example below is a bit better
```js
const addedPlugins = plugins.filter((plugin) =>
  selectedPlugins.indexOf(plugin.name) !== -1) 
```

But no need to change it, just coding style

Reply:
Still think that in this particular case is much easier to read more explicit syntax :shipit: